Modules/Quotas/test/Quotas.Domain.Tests/Tests/Identities/IdentityTests.cs

```diff
@@ -46,6 +46,52 @@ public void Can_assign_tier_quota_from_definition_to_identity()
         identity.TierQuotas.Second().Period.Should().Be(tierQuotaDefinition2.Period);
     }
 
+    [Fact]
+    public void Can_delete_individual_quota_by_id()
+    {
+        // Arrange
+        var identity = new Identity("some-address", new TierId("some-tier-id"));
+        var createdQuota = identity.CreateIndividualQuota(MetricKey.NumberOfSentMessages, 1, QuotaPeriod.Day);
+
+        // Act
+        identity.DeleteIndividualQuota(createdQuota.Id);
+
+        // Assert
+        identity.IndividualQuotas.Should().HaveCount(0);
+    }
+
+    [Fact]
+    public void Can_delete_individual_quota_by_id_with_multiple_quotas()
+    {
+        // Arrange
+        var identity = new Identity("some-address", new TierId("some-tier-id"));
+        var firstQuota = identity.CreateIndividualQuota(MetricKey.NumberOfSentMessages, 1, QuotaPeriod.Day);
+        var secondQuota = identity.CreateIndividualQuota(MetricKey.NumberOfFiles, 1, QuotaPeriod.Day);
+
+        // Act
+        identity.DeleteIndividualQuota(firstQuota.Id);
+
+        // Assert
+        identity.IndividualQuotas.Should().HaveCount(1);
+        identity.IndividualQuotas.ElementAt(0).Id.Should().Be(secondQuota.Id);
```

Comment:
You should be able to do something like this:
```suggestion
        identity.IndividualQuotas.Should().Contain(secondQuota);
```